easybuild/easyblocks/generic/intelbase.py

```diff
@@ -97,8 +94,8 @@ def __init__(self, *args, **kwargs):
         self.setup_local_home_subdir()
         self.clean_home_subdir()
 
-        if self.cfg._config['license'][0] is not None:
-            _log.deprecated('No old style license parameter, use license_file', '2.0')
+        if 'license' in self.cfg._config:
+            _log.nosupport('No old style license parameter, use license_file', '2.0')
```

Comment:
add `instead` at the end of the error message, use `"` quotes for sentences (just style, no technical reason